cmd/entrypoints/migrate.go

```diff
@@ -38,7 +40,8 @@ var migrateCmd = &cobra.Command{
 
 		if err != nil {
 			// if db does not exist, try creating it
-			pqError, ok := err.(*pq.Error)
+			cErr, ok := err.(errors2.ConnectError)
+			pqError := cErr.Unwrap().(*pgconn.PgError)
```

Comment:
is this safe to do if !ok?

Reply:
wait. should have give it a second glance. Let me fix those

Reply:
fixed.